File: modules/angular2/test/core/render/dom/events/hammer_gestures_spec.ts
import {
  AsyncTestCompleter,
  beforeEach,
  ddescribe,
  describe,
  el,
  expect,
  iit,
  inject,
  it,
  xit,
  beforeEachBindings,
  SpyObject
} from 'angular2/testing_internal';

import {HammerGesturesPlugin} from 'angular2/src/core/render/dom/events/hammer_gestures';

export function main() {
  if (typeof window !== 'undefined') {
    describe('HammerGesturesSupport', () => {

      it('should return false if Hammer.js is not loaded', () => {
        const h = new HammerGesturesPlugin();
        expect(h.supports('press')).toEqual(false);
      });

    });
  }
}

File: modules/angular2/test/core/render/dom/events/hammer_gestures_spec.dart
library angular2.render.dom.events.hammer_gestures.test;

import 'package:angular2/testing_internal.dart';
import 'package:angular2/src/core/render/dom/events/hammer_gestures.dart';

main() {
  describe('HammerGesturesSupport', () {

    it('should return false if Hammer.js is not loaded', () {
      HammerGesturesPlugin h = new HammerGesturesPlugin();
      expect(h.supports('press'), false);
    });

  });
}
